linkml/transformers/rollup_transformer.py

```diff
@@ -79,24 +77,17 @@ def transform(self, tgt_schema_name: str = None, top_class: ClassDefinitionName
         view = SchemaView(schema_copy)
         descendants = view.class_descendants(self.target_class, imports=True, mixins=True, reflexive=False, is_a=True)
         self._descendant_classes = set(descendants)
-        self._collect_slots(view)
-        return self._create_flattened_schema(schema_copy, view)
-
-    def _collect_slots(self, view: SchemaView) -> None:
-        """
-        Collect all slots from the target class and its descendant classes.
-        Populates a mapping of class name to the set of its slot names.
-        """
-        # Get induced slots for the target class.
-        target_slots = view.class_induced_slots(self.target_class)
-        self._collect_slots_for_class[self.target_class] = target_slots
```

Comment:
target_slots is used to populate `self._collect_slots_for_class`, which doesn't get used for anything ==> just remove these lines